client/components/wcc-settings-form/settings-group.js

```diff
@@ -2,42 +2,54 @@ import React, { PropTypes } from 'react';
 import CompactCard from 'components/card/compact';
 import FormSectionHeading from 'components/forms/form-section-heading';
 import SettingsItem from './settings-item';
-import { connect } from 'react-redux';
-import * as FormActions from 'state/form/actions';
-import { bindActionCreators } from 'redux';
 import SaveForm from 'components/save-form';
 
-const renderGroupItems = ( items, schema, storeOptions ) => {
-	return (
-		items.map( item => (
+const SettingsGroup = ( {
+	group,
+	schema,
+	storeOptions,
+	form,
+	saveForm,
+	dismissSuccess,
+	errors,
+} ) => {
+	const renderSettingsItem = ( item ) => {
+		const key = item.key ? item.key : item;
+		const itemErrors = errors.length ? errors.filter( ( error ) => {
```

Comment:
that's one long ternary. I think it'd be clearer to split this up. Also some comments wouldn't hurt.